apps/prairielearn/src/ee/lib/ai-grading/ai-grading.ts

```diff
@@ -127,13 +128,13 @@ export async function aiGrade({
     }
     job.info(`Calculated ${newEmbeddingsCount} embeddings.`);
 
+    const gradingJobMapping = await selectGradingJobsInfo(all_instance_questions);
+
     const instance_questions = all_instance_questions.filter((instance_question) => {
       if (mode === 'human_graded') {
         // Things that have been graded by a human
-        return (
-          !instance_question.requires_manual_grading &&
-          instance_question.status !== 'unanswered' &&
-          !instance_question.is_ai_graded
+        return (gradingJobMapping[instance_question.id] ?? []).find(
+          (job) => job.grading_method === 'Manual',
```

Comment:
Nit, `Array.prototype.some(...)` is a better fit here, and this can be made a little more concise with optional chaining:

```suggestion
        return gradingJobMapping[instance_question.id]?.some(
          (job) => job.grading_method === 'Manual',
```